api/client/ps/formatter_test.go

```diff
@@ -33,8 +38,8 @@ func TestFormat(t *testing.T) {
 				Format: "table",
 			},
 			`CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
-containerID1        ubuntu              ""                  45 years ago                                                foobar_baz
-containerID2        ubuntu              ""                  45 years ago                                                foobar_bar
+containerID1        ubuntu              ""                  24 hours ago                                                foobar_baz
+containerID2        ubuntu              ""                  24 hours ago                                                foobar_bar
```

Comment:
Is this going to be flaky (23 hours <--> 24 hours?). Would it be more stable to use 36 hours in the past, so that it would output e.g. `1 day ago`